<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clietn_fileClass</class>
 <widget class="QWidget" name="clietn_fileClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>clietn_file</string>
  </property>
  <widget class="QTextEdit" name="te_show">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>471</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>服务器ip:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_serverip">
      <property name="text">
       <string>192.168.137.1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>服务器port:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_serverport">
      <property name="text">
       <string>9000</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>481</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_connect">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>链接</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_file">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>451</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>上传文件</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="clietn_file.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_connect</sender>
   <signal>clicked()</signal>
   <receiver>clietn_fileClass</receiver>
   <slot>on_btn_connect_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>-98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_file</sender>
   <signal>clicked()</signal>
   <receiver>clietn_fileClass</receiver>
   <slot>on_btn_file_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>-52</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_btn_connect_clicked()</slot>
  <slot>on_btn_file_clicked()</slot>
 </slots>
</ui>
